projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-results/search-ai-results.component.html

```diff
@@ -125,3 +125,27 @@
     *ngIf="hasError">
   </adf-empty-content>
 </aca-page-layout>
+
+<ng-template #skeleton>
```

Comment:
Please add for this PR also some unit tests, for example to check if skeleton is visible during loading